.circleci/config.yml

```diff
@@ -422,6 +422,32 @@ jobs:
           background: true
           paths:
             - ./habitat-api
+      - run:
+          name: Build api documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:$PATH
+              
+              . activate habitat; cd habitat-api
+              python setup.py develop --all
+
+              # NOTE: Temporary workaround for above line while https://github.com/facebookresearch/habitat-api/issues/385 is unresolved
+              # Create conda env with python 3.8 to build habitat-api documentation 
+              conda create -y -n habitat-py38 python=3.8
+              . activate habitat-py38; #cd habitat-api
+              conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis
+              pip install -r requirements.txt --progress-bar off
+              
+              cd docs
+              conda install -y -c conda-forge doxygen==1.8.16
+              conda install -y  jinja2 pygments docutils
+              conda install -y -c conda-forge pelican
+              mkdir -p ../build/docs
+              ./build-public.sh
+      - save_cache:
```

Comment:
The save_cache block under "Build sim documentation" can be removed.

Reply:
Really? Don't we need it for the `Update public documentation` job?

Reply:
That save is now being done as part of this block (the `Build api documentation` block).

Reply:
Done.